include/systems/condensed_eigen_system.h

```diff
@@ -106,15 +106,49 @@ class CondensedEigenSystem : public EigenSystem
    */
   virtual std::pair<Real, Real> get_eigenpair(dof_id_type i) override;
 
+  /**
+   * \returns The name used for registration and access of the condensed A matrix.
+   */
+  static std::string condensed_matrix_A_name() { return "Condensed Eigen Matrix A"; }
```

Comment:
I don't think there needs to be public APIs for these strings, as they seem like implementation details (e.g. it's just the name that the `CondensedEigenSystem` uses to identify the matrices that it adds on the System) and users will generally call just call `get_condensed_matrix_A()` when they want to access it, right? It would be fine to store const strings on the class to avoid repeating them in multiple places, but I don't think we need to have APIs for them unless there is already a use case for that.

Reply:
That's fine - I agree.